packages/contracts-bedrock/test/kontrol/proofs/OptimismPortal.k.sol

```diff
@@ -0,0 +1,110 @@
+pragma solidity ^0.8.13;
+
+import { DeploymentSummary } from "./utils/DeploymentSummary.sol";
+import { KontrolUtils } from "./utils/KontrolUtils.sol";
+import { Types } from "src/libraries/Types.sol";
+import {
+    OptimismPortalInterface as OptimismPortal,
+    SuperchainConfigInterface as SuperchainConfig
+} from "./interfaces/KontrolInterfaces.sol";
+
+contract OptimismPortalKontrol is DeploymentSummary, KontrolUtils {
+    OptimismPortal optimismPortal;
+    SuperchainConfig superchainConfig;
+
+    function setUp() public {
+        recreateDeployment();
+        optimismPortal = OptimismPortal(payable(OptimismPortalProxyAddress));
+        superchainConfig = SuperchainConfig(SuperchainConfigProxyAddress);
+    }
+
+    function test_kontrol_in_foundry(
+        bytes[] memory _withdrawalProof,
+        Types.OutputRootProof memory _outputRootProof,
+        uint256 _l2OutputIndex,
+        Types.WithdrawalTransaction memory _tx
+    )
+        external
+    {
+        assert(optimismPortal.paused() == false);
```

Comment:
Is `assert` special in kontrol? Typically for tests I prefer `require` because you can add a custom revert message so it's easier to track down where the revert happened. Same comment on the `assert` a few lines down
```suggestion
        require(optimismPortal.paused() == false, "OptimismPortal is paused");
```

Reply:
Nothing special! You're right, I'll change the `assert`s by `requires` with a meaningful message